basyx/aas/model/aas.py

```diff
@@ -52,31 +54,38 @@ class AssetInformation:
     def __init__(self,
                  asset_kind: base.AssetKind = base.AssetKind.INSTANCE,
                  global_asset_id: Optional[base.Identifier] = None,
-                 specific_asset_id: Optional[Set[base.SpecificAssetId]] = None,
+                 specific_asset_id: Iterable[base.SpecificAssetId] = (),
                  asset_type: Optional[base.Identifier] = None,
                  default_thumbnail: Optional[base.Resource] = None):
 
         super().__init__()
         self.asset_kind: base.AssetKind = asset_kind
-        self._global_asset_id: Optional[base.Identifier] = global_asset_id
-        self.specific_asset_id: Set[base.SpecificAssetId] = set() if specific_asset_id is None \
-            else specific_asset_id
+        self.specific_asset_id: base.ConstrainedList[base.SpecificAssetId] = \
```

Comment:
Just noticed this is missing a setter for `specific_asset_id`, similar to `HasSemantics.supplemental_semantic_id`.

Reply:
Done in https://github.com/eclipse-basyx/basyx-python-sdk/pull/149/commits/6b3de298cba2ac25019124999e9f8503827ab9f1